.github/workflows/push-foundation-repo.yml

```diff
@@ -0,0 +1,17 @@
+name: Sync Cardinal-Cryptography repo with Aleph-Zero-Foundation repo
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    if: ${{ github.repository == 'Cardinal-Cryptography/aleph-node'}}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
What is the reason for `fetch-depth` set to 0?

Reply:
By default checkout@v2 fetches only one commit, so the push is not possible. We need to fetch the full history